packages/turf-union/package.json

```diff
@@ -42,7 +42,10 @@
     "write-json-file": "*"
   },
   "dependencies": {
+    "@turf/combine": "^5.1.5",
     "@turf/helpers": "^5.1.5",
+    "@turf/invariant": "^6.0.0-beta.3",
```

Comment:
@rowanwins I guess this release "breaks" stuff... 😠 

```
lerna ERR! Error [ERR_REQUIRE_ESM]: Must use @std/esm to load ES Module: /home/travis/build/Turfjs/turf/packages/turf-union/node_modules/@turf/invariant/index.mjs
```

I'll revert those `beta` releases